### DISCUSSION TITLE
Double compilation with Tailwind v3 and Webpack 4.

### DISCUSSION THREAD

**Discussion by antonbehaeghe** [tailwindlabs/tailwindcss#6528] [Help]
Hi, I've been trying to get Tailwind 3 working with my existing project, with is an ejected CRA using webpack 4.
Following the docs it seemed to be working fine, except that Tailwind wouldn't generate new classes when changing .jsx files.
Adding the watch mode to the start command solved that 👍 `cross-env TAILWIND_MODE=watch node scripts/start.js`.

But now it seems like webpack is compiling twice on change, and it is taking significantly longer before it refreshes the page.
This is not a problem per se, because it seems to be working fine.

However, I noticed that when I start webpack, and my `content` in `tailwind.config.js` is configured incorrectly and I then correct it while webpack is running, compilation is near instant and it only runs once. 

So now I'm wondering what could be causing this? Any help or insights would be greatly appreciated!

**Comment by xavier-villelegier** [comment]
Hey @antonbehaeghe ! Out of curiosity, did you find any workaround on this ? We're facing the same issue 🤔

**Reply by RobinMalfait** [reply]
Hey!

This can happen if you include the destination folder in your `content` paths because then this scenario happens:
- Make a change
- New css file written to disk
- Change in folder 
- Run webpack again (but normally should stop at this point because nothing really changed)

So double check your content paths and ensure that the output folder is not in there. Very common if you write `./**/*` for example.

Hope this helps!
